applications/source/cdc-debezium-source/src/test/docker/db2/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ibmcom/db2:11.5.4.0
```

Comment:
The OOTB DB2 database is not configured for CDC. Here are the steps one need to perform in order to enable CDC for DB2: https://debezium.io/documentation/reference/1.9/connectors/db2.html#setting-up-db2
For the MySQL and Postgres DBs there are existing CDC configured test images that we use directly, but for DB2 and Oracle, because of license restrictions, we can not pre-build and distribute such images. The closest we can do is build such on the fly.  
I've used the above Dockerfile configuration to validate (manually) that our CDC adapter works find with DB2. It helped to find some missing dependencies.